src/libs/utils/getSearch.ts

```diff
@@ -4,17 +4,23 @@ const getSearch = (
   value: string,
   contributors: StatefulContributor[]
 ): StatefulContributor[] => {
+  const normalizeText = (string: string) =>
```

Comment:
Did you add a test case for the accent problem?

Reply:
Btw, this function should be defined outside of the `getSearch` function scope to be easily tested
(not saying it must be tested but it could)

Reply:
Little simplification :

```ts
const unaccent = (string: string) =>
    string
      .normalize('NFD')
      .replace(/[\u0300-\u036f]/g, '');

const getSearch = (
  search: string,
  contributors: StatefulContributor[]
): StatefulContributor[] => {
  if (!search) return contributors;

  const searchExpression = new RegExp(
    unaccent(search).split(' ').replace(/.+/g, '(?=.*$&)').join(''),
    'gi'
  );

  return contributors.filter(({ name, intro }) => searchExpression.test(unaccent(`${name} ${intro}`)))
}
```

- Extract `unaccent` as suggested by @lutangar 
- Avoid filtering if no search by returning immediateliy
- insensitive regex instead of uppercase modification
- a bit of renaming

Reply:
There is also https://www.npmjs.com/package/remove-accents FYI

Reply:
> Did you add a test case for the accent problem?

Didn't see the accent test :p

Reply:
later ;)

Reply:
Ok I'll approve later then... :door:

Reply:
done...